apps/expert/lib/expert/engine_node.ex

```diff
@@ -28,20 +28,56 @@ defmodule Expert.EngineNode do
     @dialyzer {:nowarn_function, start: 3}
 
     def start(%__MODULE__{} = state, paths, from) do
-      this_node = inspect(Node.self())
-
-      args = [
-        "--name",
-        Project.node_name(state.project),
-        "--cookie",
-        state.cookie,
-        "--no-halt",
-        "-e",
-        "Node.connect(#{this_node})"
-        | path_append_arguments(paths)
-      ]
-
-      case Expert.Port.open_elixir(state.project, args: args) do
+      epmd_module = to_charlist(Forge.EPMD)
+
+      case :init.get_argument(:epmd_module) do
+        {:ok, [[^epmd_module]]} ->
+          :ok
+
+        _ ->
+          Application.put_env(:kernel, :epmd_module, Forge.EPMD, persistent: true)
+
+          # Note: this is a private API
+          if :net_kernel.epmd_module() != Forge.EPMD do
+            raise("""
+            you must set the environment variable ELIXIR_ERL_OPTIONS="-epmd_module #{Forge.EPMD}"
+            """)
+          end
+      end
```

Comment:
Should we do this when expert boots rather than when we try to start a child node?